implementation_1_19_4/src/main/java/de/oliver/fancynpcs/v1_19_4/attributes/CatAttributes.java

```diff
@@ -50,8 +50,21 @@ private static void setPose(Npc npc, String value) {
         Cat cat = ReflectionHelper.getEntity(npc);
 
         switch (value.toLowerCase()) {
-            case "standing" -> cat.setLying(false);
-            case "sleeping" -> cat.setLying(true);
+            case "standing" -> {
+                cat.setOrderedToSit(false);
+                cat.setInSittingPose(false, false);
+                cat.setLying(false);
+            }
+            case "sleeping" -> {
+                cat.setOrderedToSit(false);
+                cat.setInSittingPose(false, false);
+                cat.setLying(true);
+            }
+            case "sitting" -> {
+                cat.setLying(false);
+                cat.setOrderedToSit(true);
+                cat.setInSittingPose(true, false);
```

Comment:
second parameter is preventing the event from triggering so this pr also resolves that issue